static/app/components/profiling/flamegraph/flamegraphZoomViewMinimap.tsx

```diff
@@ -361,14 +381,14 @@ function FlamegraphZoomViewMinimap({
         return;
       }
 
-      const configSpaceMouse = flamegraphMiniMapView.getTransformedConfigSpaceCursor(
```

Comment:
just updated the naming to be consistent with the cursor naming we use everywhere